src/script/auth/module/action/InvitationAction.ts

```diff
@@ -38,11 +38,11 @@ export class InvitationAction {
       const invitationEmail = invitation.email?.toLowerCase();
       const alreadyInvited = inviteList.find(inviteItem => inviteItem.email.toLowerCase() === invitationEmail);
       if (alreadyInvited) {
-        const error = new BackendError({
-          code: HTTP_STATUS.CONFLICT,
-          label: BackendError.LABEL.ALREADY_INVITED,
-          message: 'This email has already been invited',
-        });
+        const error = new BackendError(
+          'This email has already been invited',
```

Comment:
Shouldn't we use translation here? There seem to be exact same entry in json file:
`"BackendError.LABEL.ALREADY_INVITED": "This email has already been invited"`

Reply:
I'm not sure this is displayed to the user (probably just shown in the console)